### PR TITLE
Galaxy Evaluation Milestone 1

### PR DESCRIPTION
**Pull request by Ali-Usama**
# External Evaluation Galaxy

The delivery is satisfactory and as per the application. However, some suggestions are mentioned in the evaluation and after that, it'll be considered as accepted.

Link to the application pull request: [Galaxy](https://github.com/w3f/Grants-Program/pull/1642)

### CODE REVIEW

**Comment by 7flash** [comment]
> # External Evaluation Galaxy
> 
> The delivery is satisfactory and as per the application. However, some suggestions are mentioned in the evaluation and after that, it'll be considered as accepted.
> 
> Link to the application pull request: [Galaxy](https://github.com/w3f/Grants-Program/pull/1642)

Updated. Ready for evaluation.

**Comment by Ali-Usama** [comment]
Thanks for the quick fixes, @7flash. I've updated my evaluation as well.

**Review by semuelle — CHANGES_REQUESTED** [review]
Thanks for the evaluation, @Ali-Usama. I had some questions about the delivery, so I'll wait for those to be resolved to merge this one. 

I also made some change suggestions to your evaluation sheet. Generally, we prefer to link to a specific version of a file or directory in these, because `HEAD` versions might change and not reflect what was reviewed in the future.

File: evaluations/galaxy_1_ali-usama.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+| 0b.    | Documentation            | <ul><li>[x] </li></ul> | [Video Tutorial](https://youtu.be/WOQvxZCiU0Q) [ReadMe](https://github.com/7flash/galaxy-polkadot/blob/d29439b081e9d71d9915b68d2ea1edfb6dd6ab7d/README.md) | Documentation is satisfactory plus the video tutorial is also helpful |
+| 0c.    | Testing & Testing Guide  | <ul><li>[x] </li></ul> | [Tests](https://github.com/7flash/galaxy-polkadot/tree/master#running-the-tests)                                      | Test cases passed. Output attached below                              |
+| 0d.    | Docker                   | <ul><li>[x] </li></ul> | [Dockerfile](https://github.com/7flash/galaxy-polkadot/blob/d29439b081e9d71d9915b68d2ea1edfb6dd6ab7d/Dockerfile)                                        | Fixed in the recent commit.                                           |
+| 1.     | Frontend: Galaxy Web App | <ul><li>[x] </li></ul> | [Src](https://github.com/7flash/galaxy-polkadot/tree/d29439b081e9d71d9915b68d2ea1edfb6dd6ab7d/src)                                                      | Features listed in the application are working fine                   |
+
+
+## General Notes
+
+## Update 4th May 2023
```
Comment (semuelle) [inline evaluations/galaxy_1_ali-usama.md:20]:
```suggestion
## General Notes

## Update 17th May 2023
Added unit tests.

## Update 4th May 2023
```

**Comment by 7flash** [comment]
Thanks @semuelle @Ali-Usama @keeganquigley @randombishop

Comments and questions received from you are highly appreciated, your well-written feedback continuously pushing project forwards, - thank you!

**Comment by RouvenP** [comment]
hi @Ali-Usama we transferred the KSM yesterday

**Pull request merged**